src/rubocop.ts

```diff
@@ -87,13 +87,22 @@ function getCurrentPath(fileUri: vscode.Uri): string {
 function getCommandArguments(fileName: string): string[] {
   let commandArguments = ['--stdin', fileName, '--force-exclusion'];
   const extensionConfig = getConfig();
+
   if (extensionConfig.configFilePath !== '') {
-    const found = [extensionConfig.configFilePath]
+    const expandedConfigFilePath = path.isAbsolute(
```

Comment:
I believe you can skip this check and just use `path.resolve(extensionConfig.configFilePath)` in all cases. [From the second example in the docs](https://nodejs.org/api/path.html#pathresolvepaths) it should be a no-op if you call `path.resolve` on path that is already absolute.